src/Audio/PlayHead.ts

```diff
@@ -233,6 +236,5 @@ export default class PlayHead {
     }
     this.gain.gain.cancelScheduledValues(0);
     this.gain.gain.linearRampToValueAtTime(0.000001, stopTime);
-    this.source.stop(stopTime);
```

Comment:
This line was causing problems with Safari during dragging a note, and is also unnecessary.